geomesa-kafka/geomesa-kafka-datastore/src/main/scala/org/locationtech/geomesa/kafka/KafkaUtils.scala

```diff
@@ -0,0 +1,150 @@
+/***********************************************************************
+* Copyright (c) 2013-2016 Commonwealth Computer Research, Inc.
+* All rights reserved. This program and the accompanying materials
+* are made available under the terms of the Apache License, Version 2.0
+* which accompanies this distribution and is available at
+* http://www.opensource.org/licenses/apache2.0.php.
+*************************************************************************/
+
+package org.locationtech.geomesa.kafka
+
+import java.io.File
+import java.nio.ByteBuffer
+import java.util.ServiceLoader
+
+import com.typesafe.scalalogging.LazyLogging
+import kafka.admin.AdminUtils
+import kafka.api.{OffsetCommitRequest, PartitionMetadata, RequestOrResponse}
+import kafka.client.ClientUtils
+import kafka.common.{OffsetAndMetadata, TopicAndPartition}
+import kafka.consumer.{ConsumerThreadId, PartitionAssignor, AssignmentContext, ConsumerConfig}
+import kafka.network.BlockingChannel
+import kafka.utils.{ZKStringSerializer, Utils}
+import org.I0Itec.zkclient.ZkClient
+import org.apache.zookeeper.data.Stat
+import org.locationtech.geomesa.kafka.consumer.Broker
+
+import scala.collection.{immutable, Map}
+
+trait AbstractKafkaUtils {
```

Comment:
[minor] maybe rename this to just KafkaUtils (if that isn't confusing with the actual kafka packages), and the have KafkaUtils08, KafkaUtils09, etc.
Same with AbstractZkUtils.